cli/Makefile

```diff
@@ -0,0 +1,41 @@
+PACKAGE := github.com/dyrector-io/dyrectorio/agent
```

Comment:
Why we call it "package" instead of "agent"?

Reply:
Removed the unnecessary lines. The file will be extended or removed when CLI will get another update (go install -able, and added to the releases) depending on our preference of global makefile.